### PR TITLE
fix(kit): overflow content in select

### LINKED ISSUE #1095: 🐞TuiSelect value text overlaps arrow icon
### Which @taiga-ui/* package(s) are the source of the bug?

kit

### Please provide a link to a minimal reproduction of the bug

https://taiga-ui.dev/components/select/API?tuiMode=null&tuiTextfieldLabelOutside=true&tuiTextfieldSize=m

### Description

Снимок экрана 2021-12-10 в 16 31 09

### Angular version

11.2.13

### Taiga UI version

2.26.0

### Which browsers have you used?

- [X] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### Which operating systems have you used?

- [X] macOS
- [ ] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

### PR DESCRIPTION
**Pull request by splincode**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [ ] Documentation content changes

Closes #1095 

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: projects/kit/components/select/select.style.less [hunk 1]
```diff
@@ -16,18 +16,21 @@
...
-.textfield {
+.t-textfield {
     border-radius: inherit;
     text-align: inherit;
 }
 
-.value {
-    display: flex;
+.t-value {
+    display: block;
```
Comment (waterplea) [inline projects/kit/components/select/select.style.less:30]: Unnecessary rule for `div`, however I'm not sure removing `flex` works here. Also adding vertical padding would probably break templates for `tuiTextfieldLabelOutside="true"` case where it's supposed to take over whole space.

Comment (splincode) [reply projects/kit/components/select/select.style.less]: `Unnecessary rule for div` yes, sorry, forgot to clean up when testing

`tuiTextfieldLabelOutside="true"` But it's working

image

**Comment by splincode** [comment]
@waterplea @nsbarsukov @nsbarsukov please, review me)

File: projects/kit/components/select/select.style.less [hunk 2]
```diff
@@ -16,18 +16,23 @@
...
 }
 
-.value {
+.t-value {
     display: flex;
     width: 100%;
     align-items: center;
+    overflow: hidden;
+
+    &__primitive {
```
Comment (waterplea) [inline projects/kit/components/select/select.style.less:35]: nit: we use single `_` symbol and we use it designate some status. Like `t-value_focused` or `t-value_error`, but not that it is nested in `t-value`. With built-in encapsulation in Angular we try to make our style sheets flat.

**Review by waterplea — APPROVED** [review]

File: projects/kit/components/select/select.style.less (hunk 2 above)
Comment (splincode) [reply projects/kit/components/select/select.style.less:35]: @waterplea that is, should I rename to `t-value-primitive`?

Comment (waterplea) [reply projects/kit/components/select/select.style.less:35]: Probably `t-primitive-value` is more correct

Comment (splincode) [reply projects/kit/components/select/select.style.less]: fixed

**Comment by splincode** [comment]
@nsbarsukov please, review me))

**Review by MarsiBarsi — APPROVED** [review]

**Review by nsbarsukov — APPROVED** [review]

**Pull request merged**